tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -10,17 +10,25 @@
 class StatusBar(PageRegion):
 
     _time_locator = (By.ID, 'statusbar-time')
+    _screen_locator = (By.ID, 'screen')
     _battery_locator = (By.ID, 'statusbar-battery')
     _mobile_connection_locator = (By.ID, 'statusbar-mobile-connection')
     _data_connection = (By.CSS_SELECTOR, '#statusbar-mobile-connection .statusbar-data')
 
     @property
     def height(self):
-        return self.root_element.rect['height']
+        if self.is_displayed:
+            return self.root_element.rect['height']
+        else:
+            return 0
 
     @property
     def is_displayed(self):
-        return self.root_element.is_displayed()
+        screen = self.marionette.find_element(*self._screen_locator)
+        if screen.get_attribute("class") == "fullscreen-app":
```

Comment:
If the fullscreen app is currently showing the status bar, the class name will be changed to "fullscreen-app minimized"